TransferMethod/Sources/AddTransferMethodController.swift

```diff
@@ -71,7 +70,7 @@ public final class AddTransferMethodController: UITableViewController {
     private lazy var infoView: UIStackView = {
         let label = UILabel()
         label.numberOfLines = 0
-
+        label.adjustsFontForContentSizeCategory = true
```

Comment:
set font and color to createAccountButton

Reply:
Font and textcolor added